### ISSUE TITLE
Build files checked into master get changed when running app

### ISSUE THREAD

**Issue by anselmbradford** [opentraildata/OpenTrails-Mobile#83]
`platforms/ios/cordova/console.log`, `platforms/ios/Trails/config.xml`, and `plugins/ios.json` are changed from what's on master when using `cordova serve ios` and `cordova emulate ios`. This is not ideal, because someone making a feature branch will have updates to the above files included in their PR unless they manually discard the changes to those files. An option is to add `platforms/*` to the .gitignore file (this was the case earlier in the project, but `/platform` hadn't been removed from master) and add `cordova platform add ios` as a step in the readme. However if this can be avoided it'd be nice to not have to build the platform directory when first cloning the repository. I'm not sure what the status is with `plugins`, can this directory be ignored in .gitignore?

**Comment by lovehandle** [comment]
Yeah, let's ignore build specific stuff. I don't think plugins can be ignored, only because it contains configurations for external dependencies.

This may be a good start: https://gist.github.com/RandyMcMillan/2408032
